test/test-reporter.mjs

```diff
@@ -0,0 +1,66 @@
+function colorize (type, text) {
+  if (type === 'pass') {
+    const whiteText = `\x1b[30m${text}`
+    // Green background with black text
+    return `\x1b[42m${whiteText}\x1b[0m`
+  }
+
+  if (type === 'fail') {
+    const blackText = `\x1b[37m${text}`
+    // Red background with white text
+    return `\x1b[41m${blackText}\x1b[0m`
+  }
```

Comment:
Can you show a screenshot of what this looks like? I am uncertain black text is going to be visible (to those of us using dark terminal backgrounds).

Reply:
Dark terminal background:
<img width="624" alt="Screenshot 2024-10-02 at 11 01 10 PM" src="https://github.com/user-attachments/assets/2c21ad9e-7ede-493d-9231-1ecd690a3f51">

White terminal background:
<img width="953" alt="Screenshot 2024-10-02 at 11 06 56 PM" src="https://github.com/user-attachments/assets/56d2cf61-a65b-4936-bc19-6bbc6dcd18e0">